client/client.go

```diff
@@ -33,7 +33,7 @@ import (
 // to built-in error instance (or in the returned error if the client is
 // configured accordingly). Certain statuses can be checked using `apistatus`
 // and standard `errors` packages. Note that package provides some helper
```

Comment:
now package doesn't provide helpers, all information is in previous sentence, so lets remove current one